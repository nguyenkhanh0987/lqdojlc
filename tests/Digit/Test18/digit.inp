100
177535168 447524717
106694949 443789614
202728211 431855882
622026088 947139089
607036270 658587040
169108220 886968627
977056327 987794383
67107747 339745313
281471588 714215794
105050366 628007373
368623690 980732733
864332065 913741137
73655624 753833421
280939173 339396821
345634700 661047806
460866702 668892698
56520013 177478908
290283588 357759160
663816557 940737410
651051763 687291302
8686373 18783129
877933358 999953230
423778503 548093054
803944751 886322225
734560385 799486498
427142626 465322975
67854319 480236306
873788869 906494685
302014625 583872331
936346766 958386911
98404322 789487958
460842419 751233377
110169407 629054717
251263205 782458922
228273436 989151442
679045250 795521555
546289850 742946944
662520535 741429870
951220584 978718217
635507134 789873764
939607447 965935413
857311191 957754656
735059304 837079256
99761426 526906909
176325687 333787417
893204281 922740158
661184425 686268850
145938046 533392579
828319019 985690825
723138892 881017595
753087324 796901902
232077471 828241166
425450440 744716476
917007722 930649347
539095548 898063237
550217739 890220397
148773520 309292579
996770962 999109979
242953246 926805824
839457770 904355127
968344272 992112312
799864154 898837656
207629076 539346037
631447567 981657894
289421878 814614465
83728618 676713243
219031446 554760765
939696349 965349721
353156618 470155012
146418116 245251196
907807934 959156483
815397765 885670425
763697293 848957875
276050541 433085216
103430283 811709674
936302289 973374891
515047844 733366740
445952937 881547970
58173693 161694388
764859420 908757412
446236166 809893805
133725162 164369607
270881488 294153216
430556893 510140126
881088698 902470789
182698166 701737525
322350628 614474039
488301047 845559950
927217284 984067826
749150230 931016207
624753105 631633957
66643010 550374422
852759170 907243062
732105479 897457368
773650482 926673810
804137950 903040447
308383588 771998674
239291368 408420732
634036137 794395750
108580465 485784240
