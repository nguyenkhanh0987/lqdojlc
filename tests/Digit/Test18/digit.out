42894621
55028678
34905370
26514296
3688395
90682857
1050323
49160027
56262963
78105841
62282303
4326792
97774934
8574942
40336961
24633527
24646906
9249630
22439559
4871440
1414853
8732937
15430438
5420136
7060757
4911057
70009626
3253778
37746167
1082246
95035657
32219192
77804393
68054898
86045339
11677281
20648782
7540970
2020587
15233435
1670965
6720577
8384467
67362751
26704826
1289935
3661395
61184795
10902816
12555996
5465678
74357912
35942737
338480
35211184
32847390
30203927
166895
79732644
6041276
2392800
7196913
48249280
28502927
62947556
88408856
49027926
1625962
16968124
18163986
1935966
5059446
6676920
23826246
96090420
2310411
22608115
45041194
20636453
12017988
40342565
5849825
5663855
11817037
2513402
73072090
39634781
35001069
3714134
14239818
371023
78252862
5124959
14352578
11516326
7199982
55247104
28797035
16277232
61757641
